marketplace/commercelayer-sku/README.md

```diff
@@ -0,0 +1,27 @@
+# Contentful UI Extension
+
+This extension lets you visually associate an SKU to any model on Contentful. The association brings the SKU `code` into the content model, so that you can make it shoppable on the front-end by leveraging the Commerce Layer API ([example](https://github.com/commercelayer/static-commerce)).
+
+![Demo](demo.gif?raw=true 'Demo')
+
+## Installation
+
+Install the extension on Contentful by adding the [extension.js](https://github.com/commercelayer/contentful-extension/blob/master/extension.json) URL on the installation dialog box:
```

Comment:
I assume this is the wrong link target..?

Reply:
Yep. It's the original directory. I'll correct.